packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -193,8 +197,8 @@ export interface NodegroupOptions {
    */
   readonly subnets?: SubnetSelection;
   /**
-   * The AMI type for your node group. If you explicitly specify the launchTemplate with custom AMI, do not specify this property, or
-   * the node group deployment will fail. In other cases, you will need to specify correct amiType for the nodegroup.
+   * The AMI type for your node group. If you explicitly specify the launchTemplate with custom AMI, either set this property to `CUSTOM` or leave
+   * it undefined, otherwise the node group deployment will fail. In other cases, you will need to specify correct amiType for the nodegroup.
```

Comment:
Just so for my own understanding, `CUSTOM` ami type is supported now if we leave it unset?